### PR TITLE
[NPU L0] Update streaming mode of example

### PR DESCRIPTION
**Pull request by plusbang**
## Description

Streaming mode of generation is used in L0 pipeline examples by default, and also provide parameter `--disable-streaming`

### 4. How to test?
- [x] Application test

### CODE REVIEW

File: python/llm/example/NPU/HF-Transformers-AutoModels/LLM/Pipeline-Models/baichuan2.py [hunk 1]
```diff
@@ -105,7 +111,7 @@ def get_prompt(message: str, chat_history: list[tuple[str, str]],
             print("input length:", len(_input_ids[0]))
             st = time.time()
             output = model.generate(
-                _input_ids, max_new_tokens=args.n_predict, do_print=True
+                _input_ids, max_new_tokens=args.n_predict, do_print=True, streamer=streamer
```
Comment (rnwang04) [inline python/llm/example/NPU/HF-Transformers-AutoModels/LLM/Pipeline-Models/baichuan2.py:114]: Let's hidden `do_print=True`

File: python/llm/example/NPU/HF-Transformers-AutoModels/LLM/Pipeline-Models/qwen.py [hunk 2]
```diff
@@ -95,7 +101,7 @@
             print("input length:", len(_input_ids[0]))
             st = time.time()
             output = model.generate(
-                _input_ids, max_new_tokens=args.n_predict, do_print=True
+                _input_ids, max_new_tokens=args.n_predict, do_print=True, streamer=streamer
             )
             end = time.time()
             print(f"Inference time: {end-st} s")
```
Comment (rnwang04) [inline python/llm/example/NPU/HF-Transformers-AutoModels/LLM/Pipeline-Models/qwen.py:107]: We can hidden output for streaming mode.

**Comment by rnwang04** [comment]
We can reduce print times from 5 to 2 or 3 for better UX.

**Comment by plusbang** [comment]
> We can reduce print times from 5 to 2 or 3 for better UX.

Sure, have updated according to comments : )

**Review by rnwang04 — APPROVED** [review]
LGTM

**Pull request merged**